src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(100);
+        ball.setColor();
```

Comment:
```suggestion
        ball.setColor(colorSupplier.getRandomColor());
```
you should call ColorSuppliers method here to set a color